DigitalLearningSolutions.Web/Views/SuperAdmin/Users/UserCentreAccounts.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.UserCentreAccounts
+@using DigitalLearningSolutions.Web.ViewComponents
+@model UserCentreAccountRoleViewModel
+@{
+    ViewData["Title"] = "UserCentreAccounts";
+}
+
+@*@section NavBreadcrumbs {
+  <partial name="~/Views/" />
+}*@
+<div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+<h1>User centre roles</h1>
+<h2>@Model.UserEntity.UserAccount.FirstName @Model.UserEntity.UserAccount.LastName (@Model.UserEntity.UserAccount.PrimaryEmail)</h2>
+
+@if (Model.CentreUserDetails.Count > 0)
+{
+  <div class="nhsuk-u-margin-bottom-6">
+  <table role="table" class="nhsuk-table-responsive">
+      <thead role="rowgroup" class="nhsuk-table__head">
+          <tr role="row">
+              <th role="columnheader" scope="col">
+                  Centre
+              </th>
+              <th role="columnheader" scope="col" class="centre-status-th">
+                  Status
+              </th>
+              <th role="columnheader" scope="col">
+                  Roles
+              </th>
+          </tr>
+      </thead>
+
+      <tbody class="nhsuk-table__body">
+      @foreach (var centreRow in Model.CentreUserDetails)
+      {
+            <tr role="row" class="nhsuk-table__row">
+                <td role="cell" class="nhsuk-table__cell choose-centre-td">
+                    <span class="nhsuk-table-responsive__heading">Centre: </span>
+                    <span class="word-break">@centreRow.CentreName</span>
+                </td>
+                <td role="cell" class="nhsuk-table__cell choose-centre-td">
+                    <span class="nhsuk-table-responsive__heading">Status: </span>
+                    <strong class="nhsuk-tag nhsuk-tag--@centreRow.Status.TagColour">@centreRow.Status.TagLabel</strong>
+                </td>
+                <td role="cell" class="nhsuk-table__cell choose-centre-td">
+                    <span class="nhsuk-table-responsive__heading">Roles: </span>
+                    <div class="centre-role-tags">
+              @if (centreRow.IsActiveAdmin)
+              {
+                          <a class="" href="">Admin</a>
+              }
+              @if (centreRow.IsDelegate)
+              {
+                          <a class="">Delegate</a>
+              }
+                    </div>
+                </td>
+            </tr>
+      }
+      </tbody>
+  </table>
+  </div>
+}
+else
+{
+        <p>You have no centre accounts.</p>
```

Comment:
Could we tweak this to read: "This user has no centre accounts."

Reply:
Review comments implemented and checked in. please review.